src/main/java/com/adq/jenkins/xmljobtodsl/dsl/strategies/DSLMethodStrategy.java

```diff
@@ -34,6 +34,11 @@ public String toDSL() {
             if (isParentAMethod) {
                 return getStrategyForObject(propertyDescriptor).toDSL();
             }
+            if (propertyDescriptor.getName().equals("healthScaleFactor")) {
+                String[] str = propertyDescriptor.getValue().split("\\.");
```

Comment:
Does Java let us cast a float to int with a builtin method?

Reply:
The propertyDescriptor value is a string, so I can't easily cast to an int and back to a string. I could do something else like this to extract the part before the decimal:
```
if (propertyDescriptor.getName().equals("healthScaleFactor")) {
                int decimalIndex = propertyDescriptor.getValue().indexOf(".");
                // Extract the part of the string before the decimal point
                String intStr = propertyDescriptor.getValue().substring(0, decimalIndex);

               String[] str = propertyDescriptor.getValue().split("\\.");
                return replaceTabs(String.format(getSyntax("syntax.method_call"),
                        methodName, intStr), getTabs());
            }
```

Reply:
The propertyDescriptor value is a string here and it looks like I can't easily cast this string to an int and then back to string without the job failing. I can do something else like this to extract everything  after and including the decimal.
```
if (propertyDescriptor.getName().equals("healthScaleFactor")) {
                // Get position of decimal point
                int decimalIndex = propertyDescriptor.getValue().indexOf(".");
                // Extract the string before the decimal point
                String intStr = propertyDescriptor.getValue().substring(0, decimalIndex);

                return replaceTabs(String.format(getSyntax("syntax.method_call"),
                        methodName, intStr), getTabs());
            }
```

Reply:
sorry I meant doing something like this:
```
String s = "0.01";
double d = Double.parseDouble(s);
int i = (int) d;
``` 
https://stackoverflow.com/questions/1450991/how-to-do-an-integer-parseint-for-a-decimal-number